client/python/gradio_client/utils.py

```diff
@@ -400,58 +398,55 @@ async def get_pred_from_sse_v1plus(
     protocol: Literal["sse_v1", "sse_v2", "sse_v2.1"],
     ssl_verify: bool,
 ) -> dict[str, Any] | None:
-    done, pending = await asyncio.wait(
-        [
-            asyncio.create_task(check_for_cancel(helper, headers, cookies, ssl_verify)),
-            asyncio.create_task(
-                stream_sse_v1plus(
-                    helper, pending_messages_per_event, event_id, protocol
-                )
-            ),
-        ],
-        return_when=asyncio.FIRST_COMPLETED,
-    )
-
-    for task in pending:
-        task.cancel()
-        try:
-            await task
-        except asyncio.CancelledError:
-            pass
+    helper.thread_complete = False
+    with concurrent.futures.ThreadPoolExecutor() as executor:
```

Comment:
Can we use the same one used in the parent `Client` to not have to create another one?